packages/experiments/src/components/Stack/docs/StackDos.md

```diff
@@ -0,0 +1,3 @@
+- Use to define the layout of components.
```

Comment:
Maybe just add "Use anywhere you would use flexbox".

Reply:
Following your previous comment, should I then just delete the don't? Because of the fact they mean the same thing.